Sources/CvTaggedSaveFormatWrapper.h

```diff
@@ -328,6 +331,9 @@ class CvTaggedSaveFormatWrapper
 #define WRAPPER_READ_CLASS_ENUM_ARRAY(wrapper,className,classType,count,name) { static int _idHint; static int _saveSeq = -1; (wrapper).ReadClassEnumArray(className "::" #name, _idHint, _saveSeq, classType,count,name); }
 #define WRAPPER_READ_CLASS_ENUM_ARRAY_ALLOW_MISSING(wrapper,className,classType,count,name) { static int _idHint; static int _saveSeq = -1; (wrapper).ReadClassEnumArray(className "::" #name, _idHint, _saveSeq, classType,count,name,true); }
 
+#define WRAPPER_READ_SHORT(wrapper,className,name)	{ static int _idHint; static int _saveSeq = -1; int val; (wrapper).Read(className "::" #name, _idHint, _saveSeq, &val); FAssert(val >= 0 ? val <= MAX_SHORT : val > -MAX_SHORT); *name = val; }
```

Comment:
This needs a comment at least to explain why its reading a short as an int, instead of just as a short. Also it should be marked for breaking save change to switch it to just reading a short directly (same for the other read/write functions)